packages/server/lambdas/database/client/index.ts

```diff
@@ -11,15 +13,21 @@ let databasePassword: string | null = null;
 
 export class DatabaseClient {
   public static async build(): Promise<PrismaClient> {
-    if (!databaseName || !databaseDomain || !databasePassword) {
-      [databaseName, databaseDomain, databasePassword] = await Promise.all([
-        getParameter(databaseNameId),
-        getParameter(databaseDomainId),
-        getSecuredStringParameter(databasePasswordId),
-      ]);
-    }
+    let DATABASE_URL: string;
+    const setLocalDatabaseConnection = environment === ENVIRONMENTS.local;
```

Comment:
should be `isLocalDatabaseConnection` ?

Reply:
Wasn't happy with set, but didn't go for is for some reason. Fixed!